src/playground-components/weather-element.html

```diff
@@ -0,0 +1,55 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/app-storage/app-localstorage/app-localstorage-document.html">
+<link rel="import" href="../../bower_components/iron-ajax/iron-ajax.html">
+<link rel="import" href="../shared-styles.html">
+<link rel="import" href="../redux-store.html">
+
+<dom-module id="weather-element">
+    <template>
+        <style>
+            #temp {
+                text-align: center;
+                font-style: italic;
+            }
+        </style>
+
+        <h3 style="text-align: center">Click the button below to fetch the current temperature in St. Louis, MO!</h3>
+        <div style="display:flex;justify-content:center;align-items:center;">
+            <paper-button id="weatherButton" raised on-click="__fetchWeather">Fetch Weather!</paper-button>
+        </div>
+        <template is="dom-if" if="{{temp}}">
+            <p id="temp">The current temperature in {{city}}{{state}} is {{temp}} degrees {{tempUnit}}! It is {{condition}} today with a wind speed of {{wind}} {{speedUnit}}.</p>
+        </template>
+
+        <iron-ajax
+            id="ajax"
+            method="GET"
```

Comment:
iron-ajax defaults to the GET method so you don't need to explicitly specify it, though some may argue the code is more readable with GET explicitly present vs. implied.